selfdrive/car/hyundai/values.py

```diff
@@ -2010,9 +2013,11 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
   CAR.GENESIS_GV80: {
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00JX1 MFC  AT USA LHD 1.00 1.02 99211-T6110 220513',
+      b'\xf1\x00JX1 MFC  AT MES LHD 1.00 1.04 99211-T6010 200514',
     ],
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00JX1_ SCC FHCUP      1.00 1.01 99110-T6100         ',
+      b'\xf1\x00JX1_ SCC -----      1.00 1.03 99110-T6000         ',
```

Comment:
`SCC_CONTROL` should come from radar which is only on A CAN afaik (bus 0), however on the swapped bus routes, it's coming from bus 1. Perhaps sending LFA instead of LKAS is the correct solution, as there is no ADAS ECU in the FW responses, thus the need for your hda2 hack above.